docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
         env_file:
             - ./.env
         ports:
-            - "80:5000"
+            - "80:80"
```

Comment:
I'm not sure this makes sense given you haven't specified which port the prod container actually should run on - if you do follow the suggestion above for the default value then make sure this matches 🙂